.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,17 +45,19 @@
 <!-- You can add additional checkboxes if it makes sense to only review parts of the code or functionality. -->
 <!-- When changes are pushed, uncheck the affected boxes. (Not all changes require full re-reviews.) -->
 
-- Code Review
-  - [ ] Review 1
-  - [ ] Review 2
-- Manual Tests
-  - [ ] Test 1
-  - [ ] Test 2
+#### Code Review
+- [ ] Review 1
+- [ ] Review 2 
+#### Manual Tests
+- [ ] Test 1
+- [ ] Test 2
 
 ### Test Coverage
-<!-- Please add the test coverage for all changes files here. You can see this when executing the tests locally (see build.gradle and package.json) or when looking into the corresponding Bamboo build plan -->
-<!-- * ExerciseService.java: 85% -->
-<!-- * programming-exercise.component.ts 95% -->
+<!-- Please add the test coverages for all changes files here. You can see this when executing the tests locally (see build.gradle and package.json) or when looking into the corresponding Bamboo build plan -->
```

Comment:
Found one small typo, but the rest looks good. Thanks for the effort 👍 
```suggestion
<!-- Please add the test coverages for all changed files here. You can see this when executing the tests locally (see build.gradle and package.json) or when looking into the corresponding Bamboo build plan. -->
```

Reply:
You can look at a text a thousand times and there will always be a typo it seems 😅

Reply:
Happens to the best of us 😄